app-backend/batch/src/main/scala/com/azavea/rf/batch/sentinel2/ImportSentinel2.scala

```diff
@@ -0,0 +1,284 @@
+package com.azavea.rf.batch.sentinel2
+
+import com.azavea.rf.batch.Job
+import com.azavea.rf.batch.util._
+import com.azavea.rf.database.{Database => DB}
+import com.azavea.rf.database.tables.{Scenes, Users}
+import com.azavea.rf.datamodel._
+import com.azavea.rf.batch.util._
+
+import io.circe._
+import io.circe.syntax._
+import cats.implicits._
+import geotrellis.proj4.CRS
+import geotrellis.slick.Projected
+import geotrellis.vector._
+import org.postgresql.util.PSQLException
+
+import java.security.InvalidParameterException
+import java.time.{LocalDate, ZoneOffset, ZonedDateTime}
+import java.util.UUID
+
+import scala.concurrent.Future
+import scala.util.{Failure, Success}
+
+case class ImportSentinel2(startDate: LocalDate = LocalDate.now(ZoneOffset.UTC))(implicit val database: DB) extends Job {
+  val name = ImportSentinel2.name
+
+  /** Required for sceneMetadata collection, to unbox Options */
+  implicit def optionToString(opt: Option[String]): String = opt.getOrElse("")
+
+  /** Get S3 client per each call */
+  def s3Client = S3(sentinel2Config.awsRegion)
+
+  def createImages(sceneId: UUID, tileInfo: Json, resolution: Float): List[Image.Banded] = {
+    val tileInfoPath = tileInfo.hcursor.downField("path").as[String].toOption.getOrElse("")
+    logger.info(s"Creating images for $tileInfoPath with resolution $resolution")
+
+    val keys = resolution match {
+      case 10f =>
+        List(
+          s3Client.getObject(sentinel2Config.bucketName, s"${tileInfoPath}/B02.jp2"),
```

Comment:
I'm not sure `getObject` is a good idea here. According to [docs](http://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/services/s3/AmazonS3Client.html#getObject-com.amazonaws.services.s3.model.GetObjectRequest-) it returns the object. However, the only thing we're doing with it in this case is getting the key string, which is the argument to the method.

If we want to check that the file at least exists, we could probably use [`doesObjectExist`](http://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/services/s3/AmazonS3.html#doesObjectExist-java.lang.String-java.lang.String-).

However, I'd be happy with just using the keys explicitly.